tests/NHSD.BuyingCatalogue.Testing.Data/SqlRunner.cs

```diff
@@ -37,5 +37,13 @@ internal static async Task<IEnumerable<T>> FetchAllAsync<T>(string selectSql, ob
                 return (await databaseConnection.QueryAsync<T>(selectSql, param).ConfigureAwait(false)).ToList();
             }
         }
+
+        internal static async Task<T> QueryFirstAsync<T>(string sql, object parameters = null)
+        {
+            using (IDbConnection databaseConnection = new SqlConnection(ConnectionStrings.GPitFuturesSetup))
+            {
+                return (await databaseConnection.QueryFirstAsync<T>(sql, parameters));
+            }
```

Comment:
Can be simplified to `using` declaration.

Reply:
👍